contracts/bonding/IBondingManager.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.11;
+
+/*
+ * @title Interface for BondingManager
+ * TODO: switch to interface type
+ */
+contract IBondingManager {
+    // External functions
+    function transcoder(uint8 _blockRewardCut, uint8 _feeShare, uint256 _pricePerSegment) external returns (bool);
+    function resignAsTranscoder() external returns (bool);
+    function bond(uint _amount, address _to) external returns (bool);
+    function unbond() external returns (bool);
+    function withdraw() external returns (bool);
+    function reward() external returns (bool);
+
+    // Public functions
+    function setActiveTranscoders() public returns (bool);
```

Comment:
I left it as public because I wasn't sure if it would ever be called internally by `BondingManager` if we change how round progression and active transcoder set selection in the future. But I do think it makes more sense for it to be marked `external` for now since it is only called by `RoundManager` externally and never internally. We can come back in the future and change it if neccessary.